[Work In Progress] Trash hint on limit hit

benschac opened the pull request:
#1281 
<!--
Please make sure you are familiar with and follow the instructions in the
contributing guidelines (found in the CONTRIBUTING.md file).

Please fill out the information below to expedite the review and (hopefully)
merge of your pull request!
-->

<!-- Is it a Bug fix, feature, docs update, ... -->
update / feature request

**What kind of change does this PR introduce?**
code change -- will add a tooltip based on users sandbox count

<!-- You can also link to an open issue here -->
https://github.com/codesandbox/codesandbox-client/issues/1281

**What is the current behavior?**
A user will hit there limit and not know why.

<!-- if this is a feature change -->
- onHover: x sandboxes in trash 

**What is the new behavior?**
- A tooltip that tells you how many sandboxes are in your trash.
<!-- Have you done all of these things?  -->

**Checklist**:

<!-- add "N/A" to the end of each line that's irrelevant to your changes -->
<!-- to check an item, place an "x" in the box like so: "- [x] Documentation" -->

- [ ] Documentation
- [ ] Tests
- [ ] Ready to be merged <!-- In your opinion, is this ready to be merged as soon as it's reviewed? -->
- [ ] Added myself to contributors table <!-- this is optional, see the contributing guidelines for instructions -->

<!-- feel free to add additional comments -->

<!-- Thank you for contributing! -->

benschac commented on packages/app/src/app/pages/Dashboard/Sidebar/TrashItem/index.js:
```diff
@@ -1,28 +1,32 @@
 
 export const entryTarget = {
-  drop: (props, monitor) => {
+  drop: monitor => {
```
N/A - just an argument that wasn't being used.

MichaelDeBoey replied:
Don't forget to change the call signature too then

SaraVieira commented:
Hey @benschac !

Thank your for contributing, need any help? :)

benschac commented:
Hello! Yes! I am having trouble getting my local environment running. Just gave it another crack. `rm -rf` 'ed and re-installed.

Using node v. 12.3.0 
yarn 1.16

this is the first error I'll get: 
```
Error running install script for optional dependency: "/Users/benjaminschachter/codesandbox-client/node_modules/fsevents, /Users/benjaminschachter/codesandbox-client/node_modules/gatsby/node_modules/webpack-dev-server/node_modules/fsevents: Command failed.
```

Updated `node-gyp` cleaned the cache from yarn, deleted node-modules. No luck.

I'll get stuff like this: 

```
node-pre-gyp WARN Tried to download(404): https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.2.4/fse-v1.2.4-node-v72-darwin-x64.tar.gz 
node-pre-gyp WARN Pre-built binaries not found for fsevents@1.2.4 and node@12.3.0 (node-v72 ABI, unknown) (falling back to source compile with node-gyp) 
```

Still google around.  Looking for a solution.

benschac commented:
I thought it was coming from a Gatsby Plugin on the site. Anyone else having this problem. Looks like `fs-modules` is getting a 404 and causing all sorts of havoc

benschac commented:
Upgraded my system, no luck. Curious, Y'all have an internal `yarn.lock` file? Maybe that could be the issue? This third party dep is out of sync?

benschac commented:
Finally got my environment to work. Downgraded to node version 8 :D. Getting build errors, but it's building woooo!

benschac commented:
```
Failed to compile.

../common/lib/utils/analytics.js
Module not found: Error: Can't resolve '../version' in '/Users/benjaminschachter/codesandbox-client/packages/common/lib/utils'
```

getting closer!

CompuIves commented:
Interesting. I think everything should work if you merge with master and run `yarn && yarn build:deps` in the root of the project.

benschac commented:
it does. This was resolved in another PR. tyty

On Tue, Jul 16, 2019 at 6:33 AM Ives van Hoorne <notifications@github.com>
wrote:

> Interesting. I think everything should work if you merge with master and
> run yarn && yarn build:deps in the root of the project.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/codesandbox/codesandbox-client/pull/1899?email_source=notifications&email_token=AATDCI44FCCYQXEDAKUENXTP7WPXPA5CNFSM4HMU47WKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD2AN2JI#issuecomment-511761701>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AATDCIZHQGRTAI6FECBPJRTP7WPXPANCNFSM4HMU47WA>
> .
>

SaraVieira commented:
Hey!

Thank you so much for your help but last week we removed all the limits for free users so this PR is no longer needed but in a good way 🎉

benschac commented:
ty. new front-end looks lovely :D
